app/models/concerns/foreman_remote_execution/host_extensions.rb

```diff
@@ -56,6 +56,7 @@ def execution_interface
 
     def remote_execution_proxies(provider, authorized = true)
       proxies = {}
+      proxies[:subnet]   = execution_interface.subnet6.remote_execution_proxies.with_features(provider) if execution_interface&.subnet6
       proxies[:subnet]   = execution_interface.subnet.remote_execution_proxies.with_features(provider) if execution_interface&.subnet
```

Comment:
I'd say the proxies from both subnet types should probably be lumped together and `uniq`ed.

Reply:
> maybe it should be a bit smarter

Full ACK to that. Unfortunately, I do not really know about the possible use-cases and how to determine them.
Could it be necessary to prefer the IPv6-proxy about the IPv4-proxy or vice versa and if so, how can we tell here :thinking: